yurt/yurt_core/utils.py

```diff
@@ -103,12 +103,13 @@ def recursive_file_modify(path, dictionary, pattern=r"%\(({0})\)s", is_dir=True)
     # Iterates through every item in the given path
     if is_dir:
         for item in os.listdir(path):
-            itempath = os.path.join(path, item)
-            # If directory, call this function again
-            if os.path.isdir(itempath) and not(itempath in [".", ".."]):
-                recursive_file_modify(itempath, dictionary, pattern)
-            else:
-                _perform_substitution(itempath, dictionary, pattern, all_vars_pattern)
+            if os.path.splitext(item)[1] not in ['.git', '.pyc']:
```

Comment:
The files that we want to be filling in the blanks to are limited to only YAML or non-compiled Python files, so potentially I could just filter for those. In any event, I just realized this is more of a bandaid solution to what was really bothering me more, which is that our `django_project` goes through a gauntlet of PyPi packaging magic as it's defined as a python package and not as an untouched data structure. That means when we do copy `django_project` over, all the compiled Python files and whatever other strange binaries that happen to exist are included.  I'll cancel this PR in favor of a better solution.